docs/topics/streamfield.rst

```diff
@@ -276,6 +276,26 @@ EmbedBlock
 
 A field for the editor to enter a URL to a media item (such as a YouTube video) to appear as embedded media on the page. The keyword arguments ``required``, ``max_length``, ``min_length`` and ``help_text`` are accepted.
 
+StaticBlock
+~~~~~~~~~~
+
+``wagtail.wagtailcore.blocks.StaticBlock``
+
+A block which doesn't have any fields, thus passes no particular values to its template during rendering. This can be useful if you need the editor to be able to insert some content which is always the same or doesn't need to be configured in Wagtail admin, such as an address, embed code from third-party services, or more complex pieces of code if the template uses template tags.
+
+This block cannot be used directly as-is. It needs to be subclassed so that you can define what will appear in Wagtail admin, by giving ``self.admin_html`` a value like in the example below:
+
+.. code-block:: python
+
+    class PostsStaticBlock(blocks.StaticBlock):
+        def __init__(self, *args, **kwargs):
+            super(PostsStaticBlock, self).__init__(*args, **kwargs)
+            self.admin_html = "<b>{}</b> - No configuration needed".format(self.meta.label)
+
+        class Meta:
+            label="Latest posts"
```

Comment:
This is missing spaces around the `=` signs. Code examples should still match the Wagtail style guide.